sys/key_rev_value.js

```diff
@@ -30,17 +30,27 @@ KRVBucket.prototype.getBucketInfo = function(restbase, req, options) {
 };
 
 KRVBucket.prototype.makeSchema = function(opts) {
-    var schemaVersionMajor = 1;
+    var schemaVersionMajor = 2;
 
-    var schema =  {
+    return {
+        // Combine option & bucket version into a monotonically increasing
+        // combined schema version. By multiplying the bucket version by 1000,
+        // we increase the chance of catching a reset in the option version.
+        version: schemaVersionMajor * 1000 + (opts.version || 0),
         options: {
-            compression: [
-                {
-                    algorithm: 'deflate',
-                    block_size: 256
-                }
-            ]
+            compression: opts.compression || [
+            {
```

Comment:
Nit: indentation: either put then together with `[]` (above and below) or indent the contents.